api/v1alpha3/types.go

```diff
@@ -23,22 +23,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// // ResourceSpec defines a generic spec that can used to define Azure resources.
-// TODO: ResourceSpec should be removed once concrete specs have been defined for all Azure resources in use.
-// type ResourceSpec interface{}
-
-// TODO: Write type tests
-
-// AzureResourceReference is a reference to a specific Azure resource by ID
-type AzureResourceReference struct {
-	// ID of resource
-	// +optional
-	ID *string `json:"id,omitempty"`
-	// TODO: Investigate if we should reference resources in other ways
-}
-
-// TODO: Investigate resource filters
```

Comment:
**resolved**: Azure doesn't have resource filters, it's an AWS concept only (Azure has resource groups).